crates/provider/src/ext/debug.rs

```diff
@@ -99,6 +136,42 @@ where
     T: Transport + Clone,
     P: Provider<T, N>,
 {
+    async fn debug_get_raw_header(&self, block: BlockNumberOrTag) -> TransportResult<Bytes> {
+        self.client().request("debug_getRawHeader", (block,)).await
+    }
+
+    async fn debug_get_raw_block(&self, block: BlockNumberOrTag) -> TransportResult<Bytes> {
+        self.client().request("debug_getRawBlock", (block,)).await
+    }
+
+    async fn debug_get_raw_transaction(&self, hash: TxHash) -> TransportResult<Bytes> {
+        self.client().request("debug_getRawTransaction", (hash,)).await
+    }
+
+    async fn debug_get_raw_receipts(&self, block: BlockNumberOrTag) -> TransportResult<Vec<Bytes>> {
+        self.client().request("debug_getRawReceipts", (block,)).await
+    }
+
+    async fn debug_get_bad_blocks(&self) -> TransportResult<Vec<Block>> {
+        self.client().request("debug_getBadBlocks", ()).await
+    }
+
+    async fn debug_trace_chain(
+        &self,
+        start_exclusive: BlockNumberOrTag,
+        end_inclusive: BlockNumberOrTag,
+    ) -> TransportResult<Vec<BlockTraceResult>> {
+        self.client().request("debug_traceChain", (start_exclusive, end_inclusive)).await
+    }
+
+    async fn debug_trace_block(
+        &self,
+        rlp_block: &[u8],
+        trace_options: GethDebugTracingOptions,
+    ) -> TransportResult<Vec<TraceResult>> {
+        self.client().request("debug_traceBlock", (rlp_block, trace_options)).await
```

Comment:
you likely want to hex encode here

Reply:
+0x prefix

Reply:
Ah, you're totally right. Not having tests bites back.